cmd/workflow-controller/main.go

```diff
@@ -173,6 +174,7 @@ func NewRootCommand() *cobra.Command {
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
 	command.Flags().IntVar(&glogLevel, "gloglevel", 0, "Set the glog logging level")
 	command.Flags().StringVar(&logFormat, "log-format", "text", "The formatter to use for logs. One of: text|json")
+	command.Flags().StringVar(&executorLogFormat, "executor-log-format", "text", "The formatter to use for executor logs. One of: text|json")
```

Comment:
Should we reuse `log-format`?

Reply:
Reusing log-format will change it for exec, and workflow-controller.

Is this desired? I am OK with this.

Reply:
Makes sense to use `log-format`, changed it.